opg-php-fpm-71-ppa-1604/Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM registry.service.opg.digital/opg-nginx-1604
+
+ENV PHP_PACKAGES="mcrypt \
+                  mongodb-clients \
+                  php-xdebug \
+                  postgresql-client \
+                  php7.1 \
+                  php7.1-cli \
+                  php7.1-curl \
+                  php7.1-fpm \
+                  php7.1-imagick \
+                  php7.1-mbstring \
+                  php7.1-mcrypt \
+                  php7.1-mongo \
+                  php7.1-pgsql \
+                  php7.1-xml"
+
+RUN apt-get install -y python-software-properties && \
```

Comment:
you would want to do an apt-update before this install to ensure you are getting the latest package